engine/battle/stats.asm

```diff
@@ -106,7 +106,10 @@ FarChangeStat:
 	jr z, .ability_check
 	cp KEEN_EYE
 	ld c, ACCURACY
-	jr nz, .check_item
+	jr z, .ability_check
+	cp MINDS_EYE
+	ld c, ACCURACY
```

Comment:
do not need to load `ACCURACY` into `c` again (you may want a comment to clarify that you still expect `c` to be `ACCURACY` here anyway even if it's not explicitly coded)